test/e2e/testdata/acme/certificate_gcp.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cert-with-acme-dns01-clouddns-ambient
+spec:
+  secretName: cert-with-acme-dns01-clouddns-ambient
+  issuerRef:
+    # The issuer created previously
+    kind: ClusterIssuer
+    name: acme-dns01-clouddns-ambient
+    dnsNames:
```

Comment:
`dnsNames` field isn't part of `spec.issuerRef`, please consider removing indent to make sure that that this becomes as `spec.dnsNames` instead!